src/test/java/org/fundacionjala/at15/pokemon/commands/CreateTrainerBattleTest.java

```diff
@@ -36,4 +37,17 @@ public void commandShouldCreateATrainerBetweenTrainerBattleJsonFile() throws Inc
                 + " trainer1 name: Ash1 trainer2 name: Ash2 isBattleOver: false";
         assertEquals(expected, Reader.getResult());
     }
+
+    @Test(expected = BattleCommandException.class)
+    public void commandShouldVerifyTheBattleCommandException() throws BattleCommandException {
+
+
+        CreateTrainerBattle trainerBattleCLI = new CreateTrainerBattle();
+        new CommandLine(trainerBattleCLI).parseArgs();
+        trainerBattleCLI.call();
+
+        /*Reader.readJson(trainerBattleCLI.getIdCreateTrainerBattle());
```

Comment:
The commented code should be eliminated if it's not necessary

Reply:
Yes you are right, thanks Adrian